### PR TITLE
[SPARK-399] install master CLI

### PR DESCRIPTION
**Pull request by mgummelt**
We're testing against master DC/OS, so for consistency, we should test against the master DC/OS CLI as well.  The immediate benefit is that this should fix the logging problem we see in the tests.

### CODE REVIEW

**Comment by mgummelt** [comment]
blocked on https://github.com/dcos/dcos-cli/pull/866

**Comment by mgummelt** [comment]
cc @susanxhuynh for review

File: bin/jenkins.sh [hunk 1]
```diff
@@ -68,14 +68,11 @@ function docker_version() {
...
-    mkdir dcos-cli
-    bash install.sh dcos-cli http://change.me --add-path no
-    source dcos-cli/bin/env-setup
-
-    # hack because the installer forces an old CLI version
-    pip install -U dcoscli
+    git clone git@github.com:dcos/dcos-cli.git
+
+    (cd dcos-cli && make env)
+    (cd dcos-cli/cli && make env)
```
Comment (susanxhuynh) [inline bin/jenkins.sh]: Should this be "cd cli" since it's already in the dcos-cli directory?

File: bin/jenkins.sh [hunk 2]
```diff
@@ -68,14 +68,11 @@ function docker_version() {
...
+    source dcos-cli/cli/bin/env-setup-dev
```
Comment (susanxhuynh) [inline bin/jenkins.sh]: What's the difference between this "env-setup-dev" vs. "env-setup"?

File: bin/jenkins.sh (hunk 1 above)
Comment (mgummelt) [inline bin/jenkins.sh]: The parens means those commands are executed in an isolated environment.  It doesn't change the CWD for subsequent commands

File: bin/jenkins.sh (hunk 2 above)
Comment (mgummelt) [inline bin/jenkins.sh]: Users use to use `env-setup` when we had them install via python instead of via the native CLI.   It's a different set of deps.

**Comment by susanxhuynh** [comment]
LGTM

**Comment by mgummelt** [comment]
retest this please

**Comment by mgummelt** [comment]
Turns out this doesn't fix the problem, because shakedown uses its own (non-master) version of the CLI.

**Pull request closed without merging**